Gruntfile.js

```diff
@@ -179,8 +179,7 @@ module.exports = function(grunt) {
 					"	},\n\n" +
 					"	render: function() {\n" +
 					'		return (\n' +
-					'			<div className="design-assets__group">\n' +
-					'				<h2><a href="/devdocs/design/gridicons">Gridicons</a></h2>\n';
+					'			<div>\n';
```

Comment:
This change is being made to accommodate this bit of feedback: https://github.com/Automattic/wp-calypso/pull/8579#discussion_r82382104